qa/suites/fs/mixed-clients/distro/ubuntu/latest.yaml

```diff
@@ -0,0 +1 @@
+.qa/distros/all/ubuntu_20.04.yaml
```

Comment:
Shaman's do additionally build for jammy, isn't it?

Reply:
I don't think so, cause if they did we should not be getting this kind of error:

```
teuthology.exceptions.VersionNotFoundError: Failed to fetch package version from https://shaman.ceph.com/api/search/?status=ready&project=kernel&flavor=default&distros=ubuntu%2F22.04%2Fx86_64&ref=testing

Reply:
Well https://shaman.ceph.com/api/search/?status=ready&project=kernel&flavor=default&distros=ubuntu%2F22.04%2Fx86_64&ref=testing does show a valid build.

Reply:
Yeah but if you look at this log http://qa-proxy.ceph.com/teuthology/yuriw-2022-08-22_21:15:31-fs-wip-yuri7-testing-2022-08-17-0943-quincy-distro-default-smithi/6986427/teuthology.log, `client.0` SHA is `none`:

```
2022-08-22T22:50:57.070 DEBUG:teuthology.task.kernel:sha1 for client.0 is None
```
`client.0` is `ceph-fuse` and i don't think have package of it for jammy

Reply:
Yeah